app/services/courses/assign_program_type_service.rb

```diff
@@ -5,13 +5,18 @@ class AssignProgramTypeService
     def execute(funding_type, course)
       case funding_type
       when 'salary'
-        course.program_type = :school_direct_salaried_training_programme
+        course.program_type = if course.provider.provider_type == 'scitt'
+                                :scitt_salaried_programme
+                              elsif course.provider.provider_type == 'university'
```

Comment:
Can we move these checks to the course/provider model? Like `university?` or `scitt?`

Reply:
Done here https://github.com/DFE-Digital/publish-teacher-training/pull/3683/commits/2fd118685d7b0778a27c80ea7e115360308be7bb